modules/enterprise/server/jar/src/main/java/org/rhq/enterprise/server/configuration/ConfigurationManagerBean.java

```diff
@@ -2718,6 +2739,20 @@ private void handlePDS(final Subject subject, Resource resource, Resource baseRe
 
     }
 
+    private Iterable<Resource> processSearchExpression(String expression, ResourceCriteria criteria,
+                                                       CriteriaQueryExecutor<Resource, ResourceCriteria> queryExecutor,
+                                                       PropertyOptionsSource.ExpressionScope expressionScope,
+                                                       Predicate<Resource> baseResourcePredicate) {
+        Iterable<Resource> foundResources;
+
+        criteria.setSearchExpression(expression);
+        foundResources = new CriteriaQuery<Resource, ResourceCriteria>(criteria, queryExecutor);
```

Comment:
Well, there's no difference to this code:

```
criteria.setSearchExpression(expression);
Iterable<Resource> foundResources = new CriteriaQuery<Resource, ResourceCriteria>(criteria, queryExecutor);
```

Now the separate definition causes some confusion if that foundResources could be loaded from elsewhere also, which isn't true (I'm assuming you've done refactoring here which is why it looks like this).